AutoAim, Gyro, MiniMap

jaewonjung6446 opened the pull request:
<!-- PR 제목은 간결하게 작성해주세요 -->
AutoAim, Gyro
## ✨ 변경 사항
<!-- 변경 사항에 대한 설명을 적어주세요 -->
1.gyro사용법
gyroSystem에서 gyro감도를 직렬화하였음. 감도 조절하여 쓰면 됩니다.(기본값 0.5f)

2.AutoAim 사용법
-AutoAim메소드는 발사버튼을 누르면 자동으로 호출됩니다(발사 버튼에 등록해야됨). 지금은 곧장 적을 똑바로 바라보게 하였으나, 발사할 때마다 조금씩 대상을 바라보게 할 예정입니다.
-Player에 있는 AutoAimSystem스크립트에서 AutoAim감지 사거리(AutoAimLength),AutoAim감지 각도(AutoAimAngle)를 조절하면 오토 에임 대상 판정 사거리, 판정 각도가 조절 가능합니다.

3.미니맵 UI
-사이즈 줌 인, 줌 아웃 기능 구현도 부가적으로 하였습니다.(변수들 직렬화 시켜놨어서 조정하시면 됩니다.)

## 📚 새로 알게 된 내용 혹은 궁금한 사항
<!-- 참고할 사항이 있다면 적어주세요 -->



## 📌 관련 이슈
<!-- 관련있는 이슈 번호(#000)을 적어주세요.
  해당 pull request merge와 함께 이슈를 닫으려면
  closed #Issue_number를 적어주세요 -->

taehyeoon committed:
Merge pull request #2 from NewSuperLightBrothers/develop

develop to main

jaewonjung6446 committed:
jaewon 브랜치가 안 불러와져서 백업 파일 커밋

1.gyro사용법
gyroSystem에서 gyro감도를 직렬화하였음. 감도 조절하여 쓰면 됩니다.(기본값 0.5f)
2.AutoAim 사용법
-AutoAim메소드는 발사버튼을 누르면 자동으로 호출됩니다. 지금은 곧장 적을 똑바로 바라보게 하였으나, 발사할 때마다 조금씩 대상을 바라보게 할 예정입니다.
-Player에 있는 AutoAimSystem스크립트에서 AutoAim감지 사거리(AutoAimLength),AutoAim감지 각도(AutoAimAngle)를 조절하면 오토 에임 대상 판정 사거리, 판정 각도가 조절 가능합니다.

taehyeoon commented on Assets/Scenes/Jaewon.unity:
메테리얼이 missing나서 핑크색으로 보임

taehyeoon commented on Assets/Scripts/Jaewon/AutoAimSystem.cs:
```diff
@@ -0,0 +1,76 @@
+
+public class AutoAimSystem : MonoBehaviour
+{
+    private Vector3 currentAim;
```
private 변수의 시작은 언더바로 시작해야 함

taehyeoon commented on Assets/Settings/URP-HighFidelity-Renderer.asset:
이 파일은 꼭 수정이 필요한 것인지?

jaewonjung6446 committed:
Merge branch 'develop' into Jaewon1

jaewonjung6446 committed:
Update Jaewon.unity

jaewonjung6446 committed:
PR용 수정

jaewonjung6446 committed:
Merge branch 'develop' into Jaewon1

jaewonjung6446 committed:
CameraMovSystem+AutoAim(CameraMovVersion)

오토 에임 시 카메라가 직접 움직이도록 하였습니다.
-_autoAimCriticalPoint가 직렬화 되어 오토 에임 시 그만큼 움직입니다.(에임이 움직이는 최대치 부여)
-한번에 대상으로 움직이지 않고, 발사 할 수록 점점 다가오는 것으로 수정
-반동 시스템을 만드려고 하는데, 다른 스크립트와 협의해야함

jaewonjung6446 committed:
MiniMapUI

미니맵 우측 상단 표시 UI
미니맵 하단의 버튼으로 축소, 확대 가능.
구체적인 수치는 MiniMapSystem에서
'YPos'-기본 줌 거리 값(기본 10)
'adjustmentValue'-줌인, 줌 아웃 시 조정되는 값(기본1)
값을 바꿔서 조정 가능.
구체적인 변경 요청 사항은 회의 때 알려주세요(미니맵이 원형이었으면 좋겠다 등등)

jaewonjung6446 committed:
UTF수정 및 오타 수정, 형식 맞추기 등등(+ReCoilSysytem 삭제)

jaewonjung6446 committed:
PR수정

Rename하면서 날라간 변수 배치

jaewonjung6446 committed:
Update Gyro_AutoAim.unity

taehyeoon reviewed (approved):
변경사항 확인
이후에 Debug.Log를 Logger.Log로 변환할 필요 있음
